Header header
float64 dist_light
string color_light